src/dev/ci_setup/setup.sh

```diff
@@ -104,6 +104,14 @@ yarnGlobalDir="$(yarn global bin)"
 export PATH="$PATH:$yarnGlobalDir"
 hash -r
 
+###
+### use the chromedriver cache if it exists
+###
+if [ -f "$dir/.chromedriver/2.42/chromedriver_linux64.zip" ]; then
+  export CHROMEDRIVER_FILEPATH="$dir/.chromedriver/2.42/chromedriver_linux64.zip"
+  export CHROMEDRIVER_SKIP_DOWNLOAD=true
```

Comment:
are those exports used for anything?

Reply:
They instruct the chromedriver post-install script to skip the download and use the version we've previously downloaded in the packer cache as described here: https://www.npmjs.com/package/chromedriver#skipping-chromedriver-download